.travis.yml

```diff
@@ -4,6 +4,7 @@
 # schema_dev gets run.
 ---
 sudo: false
+dist: precise
```

Comment:
I see, my Travis builds are all running under Precise for some reason.   Will just leave this alone, hopefully the Trusty config will catch up and mysql will work.   Otherwise the fix should be in `schema_dev` so it can be applied to all gems in the family.